src/runtime-tools/win64/onefuzz.ps1

```diff
@@ -14,6 +14,13 @@ function log ($message) {
   Write-Host -ForegroundColor Yellow $timestamp $message @args
 }
 
+function Setup-Silent-Notification {
+   # https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/registry-entries-for-silent-process-exit
+   log "installing registry key for silent termination notification of onefuzz-supervisor"
+   reg import c:\onefuzz\tools\win64\onefuzz-silent-exit.reg
```

Comment:
I see where this is committed, but maybe we should also add a script to re-generate it, with comments that would naturally document the expected contents.

Reply:
This was hand built using regedit following the above documentation for format/guidance.

Reply:
Got it. I also just looked at the file in a non-diff view, and it is more self-describing than I expected, so this is fine as-is.

Reply:
One last question about this: we set the `ReportingMode` to `0x4`, which is documented as "When silent exit is detected, a pop-up notification is displayed."

Why is that the desired behavior? It seems very oriented towards interaction with a logged-on user. Does the same setting govern some other functionality?

Reply:
If there is a local user logged in, the local user will get a popup.  It's always logged though.